Backend-API/src/main/java/com/avangers/backendapi/services/UserServiceResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.avangers.backendapi.services;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserServiceResponse {
+    private String message;
+}
```

Comment:
Oh my bad, this is what I meant. It should be called something specific. Not just response, but what kind of response it is?! 🤔